portfolio/portfolio.go

```diff
@@ -27,55 +28,74 @@ const (
 	ExchangeAddress = "Exchange"
 	// PersonalAddress is a label for a personal/offline address
 	PersonalAddress = "Personal"
+
+	defaultAPIKey = "Key"
+)
+
+var (
+	errNotETHAddress           = errors.New("not an Ethereum address")
+	errAddressIsEmptyOrInvalid = errors.New("address is empty or invalid")
+	errProviderNotFound        = errors.New("provider not found")
+	errProviderNotEnabled      = errors.New("provider not enabled")
+	errProviderAPIKeyNotSet    = errors.New("provider API key not set")
+	errPortfolioItemNotFound   = errors.New("portfolio item not found")
+	errUnsupportedCoinType     = errors.New("unsupported coin type")
```

Comment:
🐒 
* I think Coin Type is actually Coin
* And I think Coin should actually be Currency
* So could we just use `currency.ErrCurrencyNotSupported`

Reply:
Going to defer this since although Bitcoin is a crypto**currency**, it's also a coin (Bit**coin**) so are often used interchangibly (like coinmarketcap.com rather than cryptocurrencymarketcap) but I do agree that the type at the end can be shedded 👍